packages/core/src/shared/utilities/pathUtils.ts

```diff
@@ -126,3 +128,24 @@ export function getDriveLetter(path: string): string {
 
     return fullpath.substring(0, 1)
 }
+
+/**
+ * Checks if the provided path is valid and exists.
+ * @todo: Migrate fs.existsSync to fsCommon.exists
+ *
+ * @param {string} [path] - The path to be checked. If not provided, defaults to an empty string.
+ * @returns {boolean} Returns true if the path is valid and exists, otherwise returns false.
+ */
+export function isValidPath(path?: string): boolean {
+    if (isWeb()) {
+        return !!path
+    }
+
+    const fs = require('fs-extra')
```

Comment:
`existsSync` is in the normal 'fs' library. why is fs-extra needed? using plain 'fs' avoids the problem with web-mode, and then the isWeb() check on line 140 is no longer needed.
```suggestion
    const fs = require('fs')
```